dvc/command/plots.py

```diff
@@ -73,7 +73,7 @@ def run(self):
             )
 
         except DvcException:
-            logger.exception("")
+            logger.exception("failed to show plots")
```

Comment:
Sorry for a post-merge review, but we are going away from these obvious messages and deleting them everywhere. Are you sure we need it here?

Reply:
not necessarily, I added it only to make it consistent with `metrics` - I guess we could remove it in both places. How about that?

Reply:
@pared Yeah, better to remove in both cases.